source/synthDrivers/espeak.py

```diff
@@ -25,6 +25,15 @@
 	PhonemeCommand,
 )
 
+
+# A mapping of commonly used language codes to e-speak languages.

```

Comment:
In all other places in the code eSpeak is spelled "eSpeak" not "e-speak". I don't think introducing another spelling variation is a good idea.